uploader.json

```diff
@@ -3373,6 +3378,11 @@
             "glob": "/sys/class/scsi_host/host[0-9]*/fwrev",
             "symbolic_name": "scsi_fwver",
             "pattern": []
+        },
+	{
+            "glob": "/var/log/libvirt/qemu/()*.+\\.log",
```

Comment:
This spec should be changed to: `"glob": "/var/log/libvirt/qemu/*.log"`

Reply:
@bfahr As per the example mentioned [here](https://gitlab.cee.redhat.com/snippets/1265), I tried to use `/var/log/libvirt/qemu/*.log` instead of `/var/log/libvirt/qemu/().*\\.log` but it fails with the error discussed in https://stackoverflow.com/questions/3675144/regex-error-nothing-to-repeat#comment49749243_3675144